modin/experimental/engines/omnisci_on_ray/frame/data.py

```diff
@@ -323,12 +323,27 @@ def groupby_agg(self, by, axis, agg, groupby_args, **kwargs):
 
         return new_frame
 
-    def agg(self, agg):
+    def agg(self, agg, **kwargs):
+        """
+        Perform specified aggregation along columns.
```

Comment:
Maybe we shouldn't add a docstring here to avoid a conflict with #2984?

Reply:
I think that when I touch the function's signature it's my responsibility to change the doc-string as well. With changes in this commit, the `agg` doc-string from #2984 needs to be changed anyway. How do you think, should it be done here (where we have the context of changes) or in a #2984? I'd agree for any of these

Reply:
Let's keep the docstring here because new parameter `**kwargs` was added.